pkg/lightning/backend/local/duplicate.go

```diff
@@ -1074,6 +1074,37 @@ type DupeController struct {
 	importClientFactory importClientFactory
 	resourceGroupName   string
 	taskType            string
+
+	collectRemoteDupRows bool
+}
+
+// NewDupeControllerForRemoteBackend creates a new DupeController for remote backend.
+func NewDupeControllerForRemoteBackend(
+	dupeConcurrency int,
+	errorMgr *errormanager.ErrorManager,
+	splitCli split.SplitClient,
+	tikvCli *tikv.KVStore,
+	tikvCodec tikv.Codec,
+	duplicateDB *pebble.DB,
+	keyAdapter common.KeyAdapter,
+	importClientFactory importClientFactory,
+	resourceGroupName string,
+	taskType string,
+	collectRemoteDupRows bool,
```

Comment:
all those params are empty now, do you need to set them to real one later?

Reply:
Yesh, the next pr implement remote.go.